packages/react-resizable-panels-website/playwright.config.ts

```diff
@@ -1,8 +1,25 @@
 import type { PlaywrightTestConfig } from "@playwright/test";
+import {
+  createReplayReporterConfig,
+  devices as replayDevices,
+} from "@replayio/playwright";
 
 const { DEBUG } = process.env;
 
 const config: PlaywrightTestConfig = {
+  projects: [
+    {
+      name: "replay-chromium",
+      use: { ...replayDevices["Replay Chromium"] },
+    },
+  ],
+  reporter: [
+    // @ts-ignore
```

Comment:
Huh, I don't see this error locally - nor when running `pnpm run typescrip` in the root